### PR TITLE
feat(logs): remove logs from stats endpoint

### PR DESCRIPTION
**Pull request by utkarshmani1997**
This commit removes the unnecessary logs generated for getting stats via rest calls.

Signed-off-by: Utkarsh Mani Tripathi <utkarsh.tripathi@mayadata.io>

### CODE REVIEW

File: app/replica.go [hunk 1]
```diff
@@ -182,8 +182,9 @@ func startReplica(c *cli.Context) error {
 		server := rest.NewServer(s)
 		router := http.Handler(rest.NewRouter(server))
 		router = util.FilteredLoggingHandler(map[string]struct{}{
-			"/ping":          {},
-			"/v1/replicas/1": {},
+			"/ping":                   {},
+			"/v1/replicas/1":          {},
+			"/v1/replicas/1/volusage": {},
```
Comment (vishnuitta) [inline app/replica.go]: why are we adding to filteredPaths?

File: replica/rest/replica.go [hunk 2]
```diff
@@ -53,7 +53,6 @@ func (s *Server) GetReplicaStats(apiContext *api.ApiContext) *types.Stats {
 }
 
 func (s *Server) GetUsage(apiContext *api.ApiContext) (*types.VolUsage, error) {
-	logrus.Infof("GetUsage")
```
Comment (vishnuitta) [inline replica/rest/replica.go]: remove log from
`ListReplicas`
`GetReplica`
`GetReplicaStats`
`GetUsage`
`GetStats`
`GetVolUsage`

Comment (utkarshmani1997) [reply replica/rest/replica.go]: ok

File: app/replica.go (hunk 1 above)
Comment (utkarshmani1997) [reply app/replica.go]: So that logs from those paths can't be collected.

**Pull request merged**